apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -577,7 +577,7 @@ export const publishDocuments = async ({ body }, response) => {
  * @type {import('express').RequestHandler}
  */
 export const markAsPublished = async (
-	{ params, body: { publishedBlobPath, publishedBlobContainer, publishedDate } },
+	{ params, body: { publishedBlobPath, publishedBlobContainer, publishedDate = null } },
```

Comment:
Not 100% a necessity, I add it as it makes it a tad easier to read and understand this part:

`...(publishedDate && { publishedDate: new Date(publishedDate) })`

otherwise, it makes the code a bit trickier to read as there's no explicit sign that publishedDate isn't always required

Reply:
Makes sense, thanks for explaining